docs/conf.py

```diff
@@ -1,27 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
 #
-# Cleo documentation build configuration file, created by
-# sphinx-quickstart on Sun Mar  9 21:01:54 2014.
-#
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 from __future__ import annotations
 
+import datetime
 import os
+import sys
+
+
+sys.path.insert(0, os.path.abspath("../src"))
+
+import cleo
+
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
+project = "Cleo"
+author = "Sébastien Eustace"
+
+this_year = datetime.datetime.now().year
+copyright = f"{this_year}, {author}"
+
+release = cleo.__version__
+
+version_major, version_minor, _ = cleo.__version__.split(".", 2)
```

Comment:
```suggestion
version_major, version_minor, _ = release.split(".", 2)
```
since it's already a variable, let's use it